node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -45,6 +45,7 @@ object Configuration {
     .parse("rnode://de6eed5d00cf080fc587eeb412cb31a75fd10358@52.119.8.109:40400")
     .right
     .get
+  private val DefaultShardId = "rchain"
```

Comment:
I'm not sure if I agree with this as the default. I think `root` might be better. Perhaps @pyrocto would like to suggest something as well.